app/domain/mailing_lists/bulk_mail/retriever.rb

```diff
@@ -22,15 +22,17 @@ def perform
   def process_mail(mail_uid)
     imap_mail = fetch_mail(mail_uid)
 
-    validator = validator(imap_mail)
+    if imap_mail.sender_email.present?
```

Comment:
the idea of the validator is to validate e-mails.
can't we have the sender check inside validator#sender_email_valid?

https://github.com/hitobito/hitobito/blob/master/app/domain/mailing_lists/bulk_mail/imap_mail_validator.rb#L34

Reply:
Aber der Validator wird danach erst mal nach `processed_before?` befragt, und dann wird schlimmstenfalls ein Mail ohne Absender als `MailLog` persistiert. Wenn der Validator diese Aufgabe übernehmen soll, dann müsste man direkt nach (oder besser noch vor) dem `if validator.processed_before?` einen zusätzlichen Check einbauen, damit nie ein MailLog für solche fehlerhaften Mails entsteht